doc/src/gotchas.rst

```diff
@@ -784,3 +784,65 @@ from SymPy 0.6.5:
         """
         expr = Poly.cancel(powsimp(expr))
         return powsimp(together(expr.expand()), combine='exp', deep=True)
+
+
+
+Be careful about substituting for derivatives! 
+==============================================
+
+    >>> import sympy as sm
+    >>> import sympy.physics.mechanics as me
+    >>> q1, q2 = me.dynamicsymbols('q1, q2')
+    >>> a, b, c = sm.symbols('a, b, c')
+    >>> expr = q1.diff().diff()**2*q1 + q1.diff()*q2.diff()
+    >>> expr
+    q1(t)*Derivative(q1(t), (t, 2))**2 + Derivative(q1(t), t)*Derivative(q2(t), t)
+     
+.. math::
+    \displaystyle \operatorname{q_{1}}{\left(t \right)} \left(\frac{d^{2}}{d t^{2}} \operatorname{q_{1}}{\left(t \right)}\right)^{2} + \frac{d}{d t} \operatorname{q_{1}}{\left(t \right)} \frac{d}{d t} \operatorname{q_{2}}{\left(t \right)}
+
+It is worth looking at the raw form (the non-rendered form) to see what is going on:
+
+    >>> repr(expr)
+    'q1(t)*Derivative(q1(t), (t, 2))**2 + Derivative(q1(t), t)*Derivative(q2(t), t)'
```

Comment:
This is the same as the output of `expr` above (except for surrounding quotes) so it isn't adding any value to the discussion as presented.